src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -427,9 +427,9 @@ class IpexCommunicationService extends AgentService {
     await this.saveAcdcMetadataRecord(
       previousExnGrantMsg.exn.e.acdc.d,
       previousExnGrantMsg.exn.e.acdc.a.dt,
-      connectionId,
       schema.title,
-      connectionId
+      connectionId,
+      schemaSaid
```

Comment:
@Sotatek-BaoHoanga can you update the unit tests to cover this case to make sure it stays working in the future? TypeScript isn't enough since they are all strings